src/datatrove/pipeline/filters/gopher_quality_filter.py

```diff
@@ -66,10 +67,10 @@ def filter(self, doc: Document) -> bool | tuple[bool, str]:
         Returns: False if sample.text does not pass any of the the heuristic tests
 
         """
-        from nltk.tokenize import word_tokenize
-
         text = doc.text
-        words = word_tokenize(text)  # TODO we should use language id filter
+        language = doc.metadata.get("language", "en")
```

Comment:
To ensure backward compatibility, we add "en" as the default language if no language metadata is provided. Do we want to handle backward compatibility in another way?